code/src/utils/format-date/formatDate.ts

```diff
@@ -0,0 +1,28 @@
+type FormatDateOptions = {
+  locale?: string;
+  options?: Intl.DateTimeFormatOptions;
+};
+
+const formatDate = (
+  dateString: string,
+  { locale = "en-GB", options = {} }: FormatDateOptions = {}
+) => {
+  const date = new Date(dateString);
```

Comment:
I suggest extending this function so that it can also accept `Date` and not only date string, because in some cases we may need to pass `Date` directly

You can use something like this:

```jsx
const date = dateLike instanceof Date ? new Date(dateString) : dateLike;
```

Reply:
Thank you. Fixed